game/scripts/npc/heroes/witch_doctor.txt

```diff
@@ -7,6 +7,6 @@
   {
     // Abilities
     //-------------------------------------------------------------------------------------------------------------
-    "Ability13"   "special_bonus_intelligence_20" // replaces special_bonus_gold_income_20
+    "Ability13"   "special_bonus_corruption_5" // replaces special_bonus_gold_income_20
```

Comment:
Maybe a spell Amp talent instead. Or a mana regen one.

Reply:
It's not the easiest to use, but considering a huge portion of his damage comes from Death Ward and the rest comes from enhancing ally damage via Maledict, I think it actually kinda fits.

Reply:
It does fit, but I think spell amplicification would be slightly better. Armour corruption is for carry heroes imo, but it would be interesting to see.

Reply:
if his ult applied the corruption I think it's a good talent, otherwise it's meh

Reply:
Yeah its memey, but it also fits. The other talent is better (bonus casks iirc). Spell Amp talent is useless for the Death Ward for example. Mana regen is also useless, because you don't spam spells with this hero.

Reply:
darkonius could you make the corruption apply to his ulti? Else I would never personally pick up this talent